pandas/tests/reshape/test_pivot.py

```diff
@@ -1289,6 +1289,39 @@ def test_pivot_number_of_levels_larger_than_int32(self):
             df.pivot_table(index='ind1', columns='ind2',
                            values='count', aggfunc='count')
 
+    @pytest.mark.parametrize('input_vals', [
```

Comment:
Not sure why this is called `input_vals` but would be clearer if it was just `dropna`